app/api/v1/__init__.py

```diff
@@ -0,0 +1,11 @@
+# type: ignore
+from __future__ import annotations
+
+from fastapi import APIRouter
+
+from .api import router
+
+
+apiv1_router = APIRouter(tags=["API-v1"], prefix="/v1")
```

Comment:
i'm not much of a fan of prefixes at levels like this, i think one of the benefits of decorators in python is being able to put the endpoint meta info right at the function itself

this is sort of like the old pattern, where you'd make a bunch of api functions, then include them all at this (separate) router level

instead, let's put everything at the level of the function, including the full path to the api endpoint

Reply:
will do this later; want to merge this quick so you can continue work